tests/scsi/008

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
+
+. tests/zbd/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test block data lifetime support"
+QUICK=1
+
+requires() {
+	_have_fio
+	_have_scsi_debug_group_number_stats
+}
+
+submit_io() {
+	local stats_attr=/sys/bus/pseudo/drivers/scsi_debug/group_number_stats
+	echo "$1 ($3)"
+	echo "$*" >>"${FULL}"
+	local direct_io=$2
+	echo 0 > "${stats_attr}" &&
+	local fio_args wh &&
+	for wh in none short medium long extreme; do
+		if [ "${direct_io}" = 0 ]; then
+			sync
+			echo 1 > /proc/sys/vm/drop_caches
+		fi
+		fio_args=(
+			--bs=4K
+			--buffer_pattern='"'"$wh"'"'
+			--direct="${direct_io}"
+			--disable_clat=1
+			--disable_slat=1
+			--end_fsync=$((1 - direct_io))
+			--filename="${dev}"
+			--group_reporting=1
+			--gtod_reduce=1
+			--ioengine="$3"
+			--ioscheduler=none
+			--name=whint_"$wh"
+			--norandommap
+			--rw=randwrite
+			--size=4M
+			--thread=1
+			--write_hint="$wh"
+		)
+		echo "fio ${fio_args[*]}" >>"${FULL}" 2>&1
+		fio "${fio_args[@]}" >>"${FULL}" 2>&1 || return $?
+	done &&
+	grep -v ' 0$' "${stats_attr}" >> "${FULL}"
+	while read -r group count; do
+		if [ "$count" -gt 0 ]; then echo "$group"; fi
+	done < "${stats_attr}"
+}
+
+test() {
+	echo "Running ${TEST_NAME}"
+
+	local scsi_debug_params=(
+		delay=0
+		dev_size_mb=1024
+		sector_size=4096
+	)
+	_init_scsi_debug "${scsi_debug_params[@]}" &&
```

Comment:
I suggest to replace _init_scsi_debug with _configure_scsi_debug so that this test case be run with built-in scsi_debug.

Reply:
Done.